src/test/scala/uk/gov/hmrc/play/filters/CacheControlFilterSpec.scala

```diff
@@ -109,11 +111,8 @@ class CacheControlFilterSpec extends WordSpecLike with Matchers with MockitoSuga
   }
 
   "Creating the filter from config" should {
-    "load the correct values" in new WithApplication(FakeApplication(additionalConfiguration = Map("caching" -> List("image/", "text/")))) {
-      CacheControlFilter.fromConfig("caching").cachableContentTypes should be (List("image/", "text/"))
-    }
-    "throw an exception on missing config" in  {
-      an [RuntimeException] should be thrownBy CacheControlFilter.fromConfig("caching").cachableContentTypes
```

Comment:
The test never really passed in the original, one side affect of the satisfying the implicit materializer required by the 2.5 filters from play is that all tests using them must have a running application - hence the OneAppPerTest added.
A side effect of this was that that particular test stopped throwing an exception (and therefore failed) - because it was never the filter that threw the exception, but the play config code that did because there was no running application.
You can simulate exactly the same behaviour in 2.3 by simply adding WithApplication() to that test and it will start to fail. 
Therefore we figured that because test never modelled actual behaviour we should leave the code as is and remove the test.